client/dashboard/utils/site-logs-view.ts

```diff
@@ -0,0 +1,21 @@
+import { LogType, FilterType } from '../data/site-logs';
+import type { View } from '@wordpress/dataviews';
+
+const getFilterParamsFromView = ( view: View, fieldNames: string[] ): FilterType => {
```

Comment:
I think these logic don't belong in the utils directory here. Let's move them to `sites/logs/dataviews.ts` instead (or `sites/logs/dataviews/views.ts`), along with the other dataviews-related things like fields and actions.

I think it's now becoming a pattern (in the dashboard code) that we colocate dateviews-related logic together 😄

Reply:
Good point. I didn't see this as the core views functionality (not yet moved over) hence moving it to utils, but it still requires Dataviews so it should still be in a Dataviews directory. Moved in ebf0ae113b7db9433bfa4c0dffe04b8d34496561